framework/include/utils/JvarMapInterface.h

```diff
@@ -20,8 +20,19 @@ class JvarMapInterface : public T
 public:
   JvarMapInterface(const InputParameters & parameters);
```

Comment:
It is an `Interface class ("Veneer") to provide a mapping from 'jvar' in` `computeQpOffDiagJacobian into the _coupled_moose_vars array`.